core/templates/dev/head/services/LoggerService.ts

```diff
@@ -0,0 +1,24 @@
+import { ErrorHandler, Injectable } from '@angular/core';
+import {downgradeInjectable} from '@angular/upgrade/static';
+
+@Injectable()
+export class LoggerService {
+  constructor(private errorHandler: ErrorHandler) {}
+
+  log(value: any, ...rest: any[]): void {
```

Comment:
Just curious, does `...` works?

Reply:
M not sure, probably we won't have the need to use the second parameter in any case since all it does it console.log() stuff. I just implement [this.](https://github.com/angular/angular/blob/master/aio/src/app/shared/logger.service.ts)